test/metabase/models/field_values_test.clj

```diff
@@ -128,6 +128,24 @@
   (sync/sync-database! db)
   (find-values field-values-id))
 
+(deftest implicit-deduplication-test
+  (let [now (t/zoned-date-time)
+        then (t/plus now (t/millis 1))]
+    (mt/with-temp [Database    {database-id :id} {}
```

Comment:
nit: technically vars for model identifiers are deprecated and apparently meant to be removed one day. We should honor that (even if I don't believe it will happen soon)

e.g.
```suggestion
    (mt/with-temp [:model/Database    {database-id :id} {}
```